### DISCUSSION TITLE
URP（Unity Universal Render Pipeline）に関して

### DISCUSSION THREAD

**Discussion by RearAlice** [pixiv/vroid-sdk-developers#77] [Q&A] [answered]
PoolかQAなのかわからなかったのでいったんは、QAに投稿させていただきます。

現在、VRM v0.111 でURP版 MToon の実装予定（現状予定）です。
VroidSDK側もサポートする予定はありますか？

関連issue tracker：
https://github.com/vrm-c/UniVRM/issues/2027

**Comment by nyamadan** [comment, marked as answer]
UniVRMがURPに対応した後にサポートを検討いたします。申し訳ございませんがしばらくお待ちいただけますと幸いです。
